contrib/npm/install.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+const os = require("os");
+const path = require("path");
+const cTable = require("console.table");
+const archive = require("./archive");
+const {repository, version} = require("./package.json");
+
+const supportedPlatforms = [
+    {
+        TYPE: "Windows_NT",
+        ARCHITECTURE: "x64",
+        RUST_TARGET: "win64",
+        BINARY_NAME: "hurl.exe",
+        ARCHIVE_EXTENSION: ".zip",
+    },
+    {
+        TYPE: "Linux",
+        ARCHITECTURE: "x64",
+        RUST_TARGET: "x86_64-linux",
+        BINARY_NAME: "hurl",
+        ARCHIVE_EXTENSION: ".tar.gz",
+    },
+    {
+        TYPE: "Darwin",
+        ARCHITECTURE: "x64",
+        RUST_TARGET: "x86_64-osx",
+        BINARY_NAME: "hurl",
+        ARCHIVE_EXTENSION: ".tar.gz",
+    },
+    {
+        TYPE: "Darwin",
+        ARCHITECTURE: "arm64",
+        RUST_TARGET: "arm64-osx",
+        BINARY_NAME: "hurl",
+        ARCHIVE_EXTENSION: ".tar.gz",
+    },
+];
+
+function error(msg) {
+    console.error(msg);
+    process.exit(1);
+}
+
+function getPlatformMetadata() {
+    const type = os.type();
+    const architecture = os.arch();
+
+    for (let supportedPlatform of supportedPlatforms) {
+        if (type === supportedPlatform.TYPE &&
+            architecture === supportedPlatform.ARCHITECTURE
+        ) {
+            return supportedPlatform;
+        }
+    }
+    error(
+        `Platform with type "${type}" and architecture "${architecture}" is not supported by ${name}.
```

Comment:
`name` is not declared, so this is erroring for me when it gets hit.

Reply:
Good catch, I've fixed it.